src/main/kotlin/io/bazel/kotlin/builder/utils/jars/JarExtractor.kt

```diff
@@ -42,7 +43,7 @@ open class JarExtractor protected constructor(
                     Files.createDirectories(target)
                   else -> jar.getInputStream(entry).use {
                     Files.createDirectories(target.parent)
-                    Files.copy(it, target)
+                    Files.copy(it, target, StandardCopyOption.REPLACE_EXISTING)
```

Comment:
What happened here to need the `REPLACE_EXISTING` attribute?

Reply:
since we are using the same generated classes directory for all the jar creation processes during `KotlinCompile` when its trying to copy there is a `java.nio.file.FileAlreadyExistsException`. This is to avoid that.